modules/tensor_mechanics/include/materials/FiniteStrainHyperElasticViscoPlastic.h

```diff
@@ -0,0 +1,144 @@
+#ifndef FINITESTRAINHYPERELASTICVISCOPLASTIC_H
+#define FINITESTRAINHYPERELASTICVISCOPLASTIC_H
+
+#include "FiniteStrainMaterial.h"
+#include "HyperElasticStress.h"
+#include "FlowRateModel.h"
+
+/**
+ * This class is for Hyperelastic Visco-Plastic material
```

Comment:
While Doxygen might pick this up the standard position for the class documentation comment seems to be right above the actual declaration (line 22)